src/commands/info.ts

```diff
@@ -18,29 +20,25 @@ export const builder: CommandBuilder<{}, Options> = (yargs) =>
   }) as yargs.Argv<Options>;
 
 export const handler = async (argv: Arguments<Options>): Promise<void> => {
-  const { name, baseUrl } = argv;
-
-  const parts = name.split("_");
-  const [tableId, chainId] = name.split("_").reverse();
-  if (parts.length < 3) {
-    console.error(
-      "invalid table name (name format is `{prefix}_{chainId}_{tableId}`)"
-    );
-    return;
-  }
+  try {
+    const { ens, validator } = await setupCommand(argv, { readOnly: true });
 
-  const chain = parseInt(chainId);
-  const network = getChainInfo(chain);
+    let { name } = argv;
+    const [tableId, chainId] = name.split("_").reverse();
 
-  if (!network) {
-    console.error("unsupported chain (see `chains` command for details)");
-    return;
-  }
+    if (argv.enableEnsExperiment && ens) {
+      name = await ens.resolveTable(name);
+    }
+
+    const parts = name.split("_");
```

Comment:
Why are we doing this twice here? Also, isn't it still valid if there isn't a prefix?